server/base/src/main/java/org/apache/accumulo/server/master/balancer/TableLoadBalancer.java

```diff
@@ -47,8 +47,8 @@
 
   private TabletBalancer constructNewBalancerForTable(String clazzName, String table) throws Exception {
     String context = null;
-    if (null != configuration)
-      context = configuration.getTableConfiguration(table).get(Property.TABLE_CLASSPATH);
+    if (null != tabletBalancerEnvironment.getConfiguration())
```

Comment:
So I think this was done for TableLoadBalancerTest.  After taking it out and running Sunny ITs, the test spams NPEs.  It looks like since the test isn't using MAC, its generating fake data and the actual configuration is null.  I will add the check back unless you think its worth it to rewrite the tests.